### ISSUE TITLE
[BUG] Incomplete dependency chart installation after upgrading 1.0.3 to 1.1.0 and master-head.

### ISSUE THREAD

**Issue by tjjh89017** [harvester/harvester#3168]
**Describe the bug**

After upgrading Harvester from v1.0.3 to v1.1.0 or master-head, whereabouts CRD wasn't installed correctly.
Storage-network will not work properly without whereabouts CRD.
If user configure storage-network, Longhorn will not able to start instance-manager according incorrect whereabouts' crd.

**To Reproduce**
1. Install harvester v1.0.3 and upgrade to v1.1.0 or above.
2. Check Whereabouts IPPools is missing
3. `kubectl get crd | grep ippools` (we should see `ippools.whereabouts.cni.cncf.io` but we cannot)

**Expected behavior**

We have `ipools.whereabouts.cni.cncf.io` CRD

**Support bundle**

**Environment**
 - Harvester ISO version: v1.0.3 and upgrade to v1.1.0 or above
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

But we can see daemonset is installed correctly.
```
$ kubectl get daemonset -A | grep whereabouts
kube-system                harvester-whereabouts                         4         4         4       4            4           beta.kubernetes.io/arch=amd64                     4h53m
```

**Workaround**
Before configuring storage-network, please apply these [two CRDs](https://github.com/harvester/harvester/tree/v1.1.0/deploy/charts/harvester/dependency_charts/whereabouts/crds)

If you already configured storage-network, please disable storage-network and apply these [two CRDs](https://github.com/harvester/harvester/tree/v1.1.0/deploy/charts/harvester/dependency_charts/whereabouts/crds).
And re-configrue storage-network again and check if the Longhorn instance managers are ready without errors.

**Comment by tjjh89017** [comment]
add docs https://github.com/harvester/docs/pull/248

**Comment by bk201** [comment]
https://github.com/harvester/harvester/pull/3181 is a workaround. I create a [following-up issue](https://github.com/harvester/harvester/issues/3185) to find the root cause.

**Comment by TachunLin** [comment]
Verified **fixed** on `master-7c1f4a24-head` (11/22). Close this issue.

### Result
After successfully upgrade 3 nodes Harvester from v1.0.3 to latest master (11/22) 

1. Execute  `kubectl get crd ippools.whereabouts.cni.cncf.io` without errors. 
  ```
  harvester-node-0:~ # kubectl get crd ippools.whereabouts.cni.cncf.io
  NAME                              CREATED AT
  ippools.whereabouts.cni.cncf.io   2022-11-22T04:23:10Z
  ```
2. The Harvester Storage Network setting's status is `True` and the type is configured. 
  ```
  harvester-node-0:~ # kubectl get settings.harvesterhci.io storage-network -o yaml
  apiVersion: harvesterhci.io/v1beta1
  kind: Setting
  metadata:
    annotations:
      storage-network.settings.harvesterhci.io/hash: 56248882bd3ee9a76ecee5fd7253fddd42d16a55
      storage-network.settings.harvesterhci.io/net-attach-def: harvester-system/storagenetwork-pngqf
      storage-network.settings.harvesterhci.io/old-net-attach-def: ""
    creationTimestamp: "2022-11-22T04:16:48Z"
    generation: 11
    name: storage-network
    resourceVersion: "201273"
    uid: 7869639a-904e-4f48-a53b-d6636d3a037b
  status:
    conditions:
    - lastUpdateTime: "2022-11-22T06:26:21Z"
      reason: Completed
      status: "True"
      type: configured
  value: '{"vlan":1,"clusterNetwork":"mgmt","range":"192.168.0.0/24"}'
  ```
3. Longhorn instance-managers are working. 
  ```
  harvester-node-0:~ # kubectl get pods -n longhorn-system -l longhorn.io/component=instance-manager -o yaml
  apiVersion: v1
  items:
  - apiVersion: v1
    kind: Pod
    metadata:
      annotations:
        cni.projectcalico.org/containerID: 3a7f390982daf111eec3f6ffed3977b785362f19a3467f5461dc243744531190
        cni.projectcalico.org/podIP: 10.52.1.110/32
        cni.projectcalico.org/podIPs: 10.52.1.110/32
        k8s.v1.cni.cncf.io/network-status: |-
          [{
              "name": "k8s-pod-network",
              "ips": [
                  "10.52.1.110"
              ],
              "default": true,
              "dns": {}
          },{
              "name": "harvester-system/storagenetwork-pngqf",
              "interface": "lhnet1",
              "ips": [
                  "192.168.0.6"
              ],
              "mac": "22:0f:21:19:92:a8",
              "dns": {}
          }]
        k8s.v1.cni.cncf.io/networks: '[{"namespace": "harvester-system", "name": "storagenetwork-pngqf",
          "interface": "lhnet1"}]'
        k8s.v1.cni.cncf.io/networks-status: |-
          [{
              "name": "k8s-pod-network",
              "ips": [
                  "10.52.1.110"
              ],
              "default": true,
              "dns": {}
          },{
              "name": "harvester-system/storagenetwork-pngqf",
              "interface": "lhnet1",
              "ips": [
                  "192.168.0.6"
              ],
              "mac": "22:0f:21:19:92:a8",
              "dns": {}
          }]
        kubernetes.io/psp: global-unrestricted-psp
        longhorn.io/last-applied-tolerations: '[{"key":"kubevirt.io/drain","operator":"Exists","effect":"NoSchedule"}]'
      creationTimestamp: "2022-11-22T06:26:24Z"
  
  ```
4. Create and start a VM without errors.
  ![image](https://user-images.githubusercontent.com/29251855/203248598-942556ec-fd40-40e3-bea9-89b4f2eac594.png)

### Test Information
* Test Environment: 3 nodes Harvester remote kvm machines
* Harvester version: master-7c1f4a24-head

### Verify Steps

1. Install harvester v1.0.3 with 3 nodes.
2. Upgrade to master-head with this PR.
3. Execute kubectl get crd ippools.whereabouts.cni.cncf.io without errors. (We need to have this CRD)
4. Create and setup storage-network in the harvester without errors.
5. Check Longhorn instance-managers are working.
6. Create and start a VM without errors.
